modules/metering-upgrade-verify.adoc

```diff
@@ -0,0 +1,97 @@
+// Module included in the following assemblies:
+//
+// * metering/metering-upgrading-metering.adoc
+
+[id="metering-upgrade-verify_{context}"]
+= Verifying Metering Upgrade
+
+You can verify the metering upgrade by performing any of the following:
+
+*  Check the Metering Operator ClusterServiceVersion (CSV) for the new metering version through either the web console or by using the `oc` CLI.
+**  Web console:
++
+.Procedure
```

Comment:
Whole lotta nesting going on. :) When I've included both methods, I usually just write `.Procedure (UI)` and `.Procedure (CLI)`. Seems easier to scan. (And this may negate the need for the anchor links I mentioned above.)